app/src/main/res/values/strings.xml

```diff
@@ -1866,8 +1866,8 @@
     <string name="pocket_stories_categories_header">Stories by topic</string>
     <!-- Text of a button allowing users to access an external url for more Pocket recommendations. -->
     <string name="pocket_stories_placeholder_text">Discover more</string>
-    <!-- Title of an app feature. Smaller than a heading.-->
-    <string name="pocket_stories_feature_title">Powered by Pocket.</string>
+    <!-- Title of an app feature. Smaller than a heading. The first parameter is product name Pocket -->
+    <string name="pocket_stories_feature_title">Powered by %s.</string>
```

Comment:
Same as above.

Reply:
Good catch! this has already been exposed on Pontoon so needs new ID :) thanks!